tests/chat_model/test_mistral_chat_model.py

```diff
@@ -90,6 +96,33 @@ def minus(a: int, b: int) -> int:
     assert len(list(message.content)) == 2
 
 
+@pytest.mark.mistral
+def test_mistral_chat_model_few_shot_prompt():
+    class Quote(BaseModel):
+        quote: str
+        character: str
+
+    chat_model = MistralChatModel("mistral-large-latest")
+    message = chat_model.complete(
+        messages=[
+            SystemMessage("You are a movie buff."),
+            UserMessage("What is your favorite quote from Harry Potter?"),
+            AssistantMessage(
+                Quote(
+                    quote="It does not do to dwell on dreams and forget to live.",
+                    character="Albus Dumbledore",
+                )
+            ),
+            # Mistral requires AssistantMessage after tool output
+            # TODO: Automatically add this in ChatModel.complete like for tool calls
+            AssistantMessage("."),
```

Comment:
@mnicstruwig FYI, should only be needed if doing few-shot prompting.

The OpenAI and Mistral APIs are starting to diverge so I've made an issue to switch to using the Mistral python client. https://github.com/jackmpcollins/magentic/issues/280